great_expectations/init_notebooks/pandas/validation_playground.ipynb

```diff
@@ -48,7 +50,9 @@
    "source": [
     "## 2. List the CSVs in your folder\n",
     "\n",
-    "The `DataContext` will now introspect your pandas `Datasource` and list the CSVs it finds. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#data-assets)"
+    "The `DataContext` will now introspect your pandas `Datasource` and list the CSVs it finds. \n",
+    "\n",
+    "[Read more in the tutorial](https://docs.greatexpectations.io/en/latest/tutorials/validate_data.html?utm_source=notebook&utm_medium=validate_data#list-data-assets)"
```

Comment:
I intentionally compacted these newlines to make the notebook less overwhelming. Shorter is better.

Reply:
+1